rickshaw-run

```diff
@@ -2168,12 +2168,12 @@ sub process_roadblocks() {
     if (defined $messages_ref) {
         foreach my $msg (@{ $$messages_ref{'received'} })  {
             if (exists $$msg{'payload'}{'message'}{'user-object'}{'new-followers'}) {
-                @new_followers = @{ $$msg{'payload'}{'message'}{'user-object'}{'new-followers'} };
-                printf "Endpoint created followers: %s\n", join(" ", sort(@new_followers));
-                last;
```

Comment:
I suppose you meant to drop this `last` call?

Reply:
Yes, absolutely.  I am not even sure why it was there in the first place, but now more than ever we have to process extra followers from all endpoints.